engage/app/styles/pages.less

```diff
@@ -46,3 +45,11 @@ a.list-group-item .list-group-item-heading.link-item-name {
 .media-item {
   padding: 15px;
 }
+/* calendar page */
+
+.calendar-list-item {
+  color: @theme2;
+  border-top: none;
+  border-left: none;
+  border-right: none;
```

Comment:
This is the same as the `engage-list-item` class. We made it for a reason, use it on these guys